spec/factories/container_deployment_node.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :container_deployment_node, :class => "ContainerDeploymentNode" do
+  end
```

Comment:
consider adding the sequence number for the deployment nodes:
`sequence(:name) { |n| "container_deployment_node_#{seq_padded_for_sorting(n)}" }`